src/exchange/providers/openai.py

```diff
@@ -43,7 +43,7 @@ def from_env(cls: Type["OpenAiProvider"]) -> "OpenAiProvider":
                 "Failed to get OPENAI_API_KEY from the environment, see https://platform.openai.com/docs/api-reference/api-keys"
             )
         client = httpx.Client(
-            base_url=url,
+            base_url=url + "v1/",
```

Comment:
how does this work today? don't know how it could if this is what is needed?

Reply:
integration tests pass both openai and ollama, and after this azure does too! that's how we know how it works: https://github.com/square/exchange/pull/60/files

Reply:
the other way we know it is fine is that the VCR cassettes didn't cry or need to change. If the path changed, they would have failed.

Reply:
but if by "today" you meant before this, the `_post` call used to be `self.client.post("v1/chat/completions"` and is now `self.client.post("chat/completions"`

Reply:
ah - I needed to scroll just a bit more...